src/com/javabasic/lesson14/homework/coffeemaker/Americano.java

```diff
@@ -0,0 +1,21 @@
+package com.javabasic.lesson14.homework.coffeemaker;
+
+public class Americano extends Drink {
+    private double cost = 1.5;
+    private int count;
+
+    public Americano(int count) {
+        this.count = count;
+    }
+    public double getCost() {
```

Comment:
this getter misleads, if we have a field cost method getCost should return just cost.
If we do some calculation, better to chose another name to highlight that we calculate something new here.